README.md

```diff
@@ -9,11 +9,15 @@ It is the first test version developed for the BACKEND-DEVELOPMENT course taught
 ## Initialize database and migrations
 #### before run the app, please run below command
 
-    flask db init - this will create a folder inside the app directory.
+`flask db init` -  this will create a folder inside the app directory.
     
-    flask db migrate - this will create all the migrations.
+`flask db migrate` - this will create all the migrations.
     
-    flask db upgrade
+`flask db upgrade`
+
+## How To Run    
+For running Nemory, type the following:
+`python main.py`
```

Comment:
Please specify how to run a program

Reply:
And also missing pip install -r requirements.txt

Reply:
There might be other tools than pip for package management in this project, so its easier to have you tell me in the README how i should run it.